lib/Mail/DKIM/ARC/Seal.pm

```diff
@@ -83,6 +83,7 @@ sub new {
     $self->timestamp(
         defined $prms{'Timestamp'} ? $prms{'Timestamp'} : time() );
     $self->expiration( $prms{'Expiration'} ) if defined $prms{'Expiration'};
+    $self->tags( $prms{'Tags'} ) if defined $prms{'Tags'};
```

Comment:
Should this be 'SealTags'? In the Pod in `Signer.pm` it states Tags are for the signature and SealTags are for the seal.

Reply:
No, that's right, It's Tags internally to each of the ARC::Seal or ARC::MessageSignature classes, but the ARC::Signer class passes them through, SealTags becomes Tags here when creating a new ARC::Seal

$self->{'SealTags'} ? ( Tags => $self->{'SealTags'} )